/**:
  ros__parameters:
    class_names: ["CAR", "PEDESTRIAN", "BICYCLE"]
    point_feature_size: 7  # x, y, z, timelag and car, pedestrian, bicycle
    max_voxel_size: 40000
    point_cloud_range: [-76.8, -76.8, -3.0, 76.8, 76.8, 5.0]
    voxel_size: [0.32, 0.32, 8.0]
    downsample_factor: 1
    encoder_in_feature_size: 12
    yaw_norm_thresholds: [0.3, 0.0, 0.3]
